dnsmasq.template

```diff
@@ -0,0 +1,25 @@
+# Template file for dnsmasq generation
+# dnsmasq will default to sending v4 + v6 address of dnsmasq server
+
+# Enforce some standards
+bogus-priv
+domain-needed
+
+# Defaults on the pi
+dhcp-mac=set:client_is_a_pi,B8:27:EB:*:*:*
+dhcp-reply-delay=tag:client_is_a_pi,2
+
+# Mannually add all the interfaces you want dnsmasq to run on
+# interface=eth0
```

Comment:
I'm a bit confused by this part... shouldn't all the right interfaces get filled in by the dhclient-ipv6 script?

Reply:
Good catch. Will fix.